src/sentry/integrations/cloudflare/metadata.py

```diff
@@ -1,17 +1,23 @@
 from __future__ import absolute_import
 
+import logging
 import six
 
 from rest_framework.response import Response
 
 from sentry.api.base import Endpoint
 from sentry.api.bases.user import UserPermission
 
+logger = logging.getLogger('sentry.integrations.cloudflare')
+
 
 class CloudflareMetadataEndpoint(Endpoint):
     permission_classes = (UserPermission, )
 
     def get(self, request):
+        logger.info('cloudflare.metadata', extra={
+            'user_id': six.text_type(request.user.id),
```

Comment:
This can just stay an integer for logging. It's typed as an integer in ES anyways.